### ISSUE TITLE
How to go back to macOS

### ISSUE THREAD

**Issue by WebHeer** [AsahiLinux/docs#44]
I tested Asahi Linux on my M1 Mini. Now I want to go back to macOS but that doesn't work. The Mac partition is 60GB in size (to small to instal macOS) and 940GB of empty space that I can no longer approach.

How can I format the disk in such a way so that I can install macOS again.

Anyone an idea how I can do that?
![IMG_0184](https://user-images.githubusercontent.com/66496665/197415930-a8c30e6e-09b1-42b2-8ec3-1ff619f77acd.JPEG)

**Comment by 0neP1us** [comment]
I Might Be Incorrect But, I Have Heard You Just Delete The Linux Partitions And Resize Apple Partition.

**Comment by WebHeer** [comment]
Unfortunately I can no longer enlarge the partitions. As you can see in the picture, the 1TB disk only has 524.3MB + 59.45GB and 5.37GB available. So 940GB of the disk has become unreachable. I want it back and then install macOS back on it.

**Comment by WebHeer** [comment]
Unfortunately this entry is closed, but still I can't reach the 940GB no matter what I try with 'diskutil'. Can someone give me the correct instructions, so that I have access to the entire hard disk again?

**Comment by marcan** [comment]
What have you tried, exactly? Please post the output of `diskutil list`.

**Comment by WebHeer** [comment]
Here is a screenshot of the current situation. There is now a freespace off 994,7 free space.
![IMG-0237](https://user-images.githubusercontent.com/66496665/198052686-326ef0b8-ac4a-415f-9be8-76205d3d4be0.jpg)

**Comment by R8s6** [comment]
> You've deleted all the OS partitions, so just create a new one with `diskutil addPartition disk0s1 apfs macOS 0`.

Thank you so much! i was in the same boat as @WebHeer (that I had removed the main macOS `disk0s2` partition), i googled everywhere and try `...addPartition disk0s2` thinking `disk0s2` being the new partition, which did not work.

I went on the path to create a macOS usb from windows/linux which required a lot of other messy staff. Then I saw this comment and it solved my headache instantly, thank you so so much again!
